src/renderer/corporate/common/corporate-shareholding.jsx

```diff
@@ -110,12 +110,10 @@ ref={c => {
 
 const CorporateShareholding = withStyles(styles)(props => {
 	const { classes, members = [] } = props;
-	const shareholders = members.filter(m => m.positions.find(p => p.position === 'shareholder'));
-
+	const shareholders = members.filter(m => m.identity.positions.find(p => p === 'shareholder'));
```

Comment:
Is it really only for shareholders? I can see this working for other equity types with different name, depending on the title attribute in equity for the meber:

```
"member": {
"type": "object",
"properties": {
"position": {
"title": "Member",
"description": "Co-owner of a business, who oversees and runs the that business.",
"const": "member"
},
"equity": {
  "title": "Membership Interest",
  "$ref": "#/definitions/equity"
}
}
},
```

Reply:
this probably can be moved to https://github.com/SelfKeyFoundation/Identity-Wallet/issues/1865